sample_workloads/nccltest/a3-mega/README.md

```diff
@@ -0,0 +1,148 @@
+# TCPX NCCL Level Benchmarks
```

Comment:
Maybe use the external facing name for the A3+ transport?

Reply:
+1, "GPUDirect-TCPXO"